index.ts

```diff
@@ -84,45 +77,37 @@ class Repeater {
                     }
                     return true;
                 }
-            );
-        if (options.intervalMode === IntervalMode.fixed) {

```

Comment:
why deleted?

Reply:
baka dati pa ung mga changes
![image](https://github.com/user-attachments/assets/cdaf95b6-bd42-40da-80df-2ceadcf26575)
dito rin kse sa commit nare is wala ung `const chain = () =>` so baka mas dati pa, pati ung `options.intervalMode` wala din

Reply:
bruh